spec/integration/aws_vpc_spec.rb

```diff
@@ -194,5 +194,25 @@
         end
       end
     end
+
+    with_aws "with no other objects" do
+      context "When operating in different regions" do
+        let(:regions) { %w(us-west-1 us-west-2) }
+
+        it "creates both VPCs" do
+          expect_recipe {
+            regions.each_with_index do |region, idx|
+              with_driver "aws::#{region}" do
```

Comment:
I actually don't know.  https://github.com/chef/chef-provisioning-aws/blob/3e52a6f56c8b5bdff6334730e9588532b0bf649c/spec/aws_support.rb#L84-L105 stores a driver reference that we use in the `purge_all` method in that class.  But I don't know off the top of my head how this creates the driver used by the resources...

Does your test pass or fail right now?  If it fails, does it fail with the `AWS::EC2::Errors::InvalidVpcID::NotFound` you originally reported?  Can you get a full stacktrace?